pkg/deploy/api/v1/conversion.go

```diff
@@ -48,7 +49,10 @@ func convert_api_DeploymentTriggerImageChangeParams_To_v1_DeploymentTriggerImage
 	return nil
 }
 
-func convert_v1_RollingDeploymentStrategyParams_To_api_RollingDeploymentStrategyParams(in *RollingDeploymentStrategyParams, out *newer.RollingDeploymentStrategyParams, s conversion.Scope) error {
+func Convert_v1_RollingDeploymentStrategyParams_To_api_RollingDeploymentStrategyParams(in *RollingDeploymentStrategyParams, out *newer.RollingDeploymentStrategyParams, s conversion.Scope) error {
+	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
```

Comment:
Same comment about calling the exact function.  These are going to be copy/pasted, so it pays to have it how we want it.